obspy/io/mseed/headers.py

```diff
@@ -581,11 +497,11 @@ class MSTraceSeg(C.Structure):
     ('samprate', C.c_double),         # Nominal sample rate (Hz)
     ('samplecnt', C.c_int64),         # Number of samples in trace coverage
     ('datasamples', C.c_void_p),      # Data samples, 'numsamples' of type
-                                      # 'sampletype'
+    # 'sampletype'
```

Comment:
This is wrapped from the previous line; looks pretty weird unindented like this.